### PR TITLE
ADDED: Neural Networks and Deep Learning

### PR DESCRIPTION
**Pull request by kchatr**
Added a course for machine learning for Issue #3

### CODE REVIEW

File: courses/data.js [hunk 1]
```diff
@@ -38,4 +38,12 @@ let data = [
     image: "intro-linux.png",
     tags: ["linux", "bash"],
   },
+  {
+    name: "Neural Networks and Deep Learning",
+    instructor: "Andrew Ng",
+    url: "https://www.coursera.org/learn/neural-networks-deep-learning",
+    description: "If you want to break into cutting-edge AI, this course will help you do so. Deep learning engineers are highly sought after, and mastering deep learning will give you numerous new career opportunities. Deep learning is also a new 'superpower' that will let you build AI systems that just weren't possible a few years ago.",
+    image: "",
+    tages: ["machine learning", "ai", "deep learning", "neural networks"]
```
Comment (ishandeveloper) [inline courses/data.js:47]: Please fix this typo `tages`, and write `tags` instead.

**Review by ishandeveloper — CHANGES_REQUESTED** [review]
Please take a look at the comments I've made and resolve those issues.

**Review by kchatr — COMMENTED** [review]
Corrected the typo and added comma to the end for future contributors.

**Comment by ishandeveloper** [comment]
Congrats on submitting your first PR! 🎊
Your changes have been merged and should be reflected on the website in a few minutes. 

### Happy hacking!

**Comment by kchatr** [comment]
Thank you @ishandeveloper!! It's actually my first ever Hacktoberfest contribution :D.

**Comment by ishandeveloper** [comment]
Hey @kchatr ! Just came back to see, I forgot to properly greet you on your first-time contribution towards an open-source project. :) 

Thank you for giving this repository the opportunity to help you guide in your first contribution. On behalf of everyone here, I'd like to say, 

### Welcome to the open-source community!
Have fun hacking!

**Pull request merged**